microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -366,6 +366,41 @@ If you find yourself running the same debug code in the **Console** tool over an
 See [Run snippets of JavaScript on any webpage](./snippets.md).
 
 
+<!-- ====================================================================== -->
+## Use `# sourceURL` to name evaluated and inline code in the Sources tool
+
+The `# sourceURL` pragma, with syntax `//# sourceURL=my-assigned-file-name.js` or `/*# sourceURL=my-assigned-file-name.css*/`, is a special comment that makes it easier to work with evaluated and inline code in the **Sources** tool.
```

Comment:
"makes it easier" - ok, but vague.  Give UI specifics.  Where is the file name missing, if you don't do this?  Where is the file name shown, if you do this?